cli/src/rust_template.rs

```diff
@@ -18,6 +18,8 @@ use std::{
     process::Stdio,
 };
 
+const ANCHOR_MSRV: &str = "1.83.0";
```

Comment:
The next version of anchor will require Rust 1.89 due to https://github.com/solana-foundation/anchor/pull/3842